src/assets/icons/icon_done.tsx

```diff
@@ -0,0 +1,3 @@
+export const iconDone = <svg width="51" height="42" viewBox="0 0 51 42" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think, better if we try to use width/height/color as props for all svg icons. With these props we can change size or fill color for every svg icon outside. 
Check, please, in another places too.

Reply:
Fix this maybe later.
I think simple set width in css and icon resize proportional.